lib/Renard/Curie/Data/PDF.pm

```diff
@@ -2,43 +2,101 @@ use Modern::Perl;
 package Renard::Curie::Data::PDF;
 
 use Capture::Tiny qw(capture_stdout);
+use XML::Simple;
+use Path::Tiny;
+use Alien::MuPDF 0.004;
 
-sub mudraw_get_pdf_page_as_png {
-	my ($pdf_filename, $pdf_page_no) = @_;
+BEGIN {
+	our $MUTOOL_PATH =  path(Alien::MuPDF->dist_dir, qw(bin mutool));
+}
 
+sub _call_mutool {
+	my @args = ( $Renard::Curie::Data::PDF::MUTOOL_PATH, @_ );
 	my ($stdout, $exit) = capture_stdout {
-		system("mudraw",
-			qw( -F png ),
-			qw( -o -),
-			$pdf_filename,
-			$pdf_page_no,
-		);
+		system( @args );
 	};
 
-	die "Unexpected mudraw exit: $exit" if $exit;
+	die "Unexpected mutool exit: $exit" if $exit;
+
+	return $stdout;
+}
+
+sub get_mutool_pdf_page_as_png {
+	my ($pdf_filename, $pdf_page_no) = @_;
+
+	my $stdout = _call_mutool(
+		qw(draw),
+		qw( -F png ),
+		qw( -o -),
+		$pdf_filename,
+		$pdf_page_no,
+	);
+
+	return $stdout;
+}
+
+sub get_mutool_text_stext_raw {
```

Comment:
This calls `mutool draw`, but gets an XML representation of the the text on the page.